qt/scientific_interfaces/Inelastic/Common/RunWidget/RunView.cpp

```diff
@@ -23,9 +23,11 @@ void RunView::subscribePresenter(IRunPresenter *presenter) { m_presenter = prese
 
 void RunView::notifyRunClicked() { m_presenter->handleRunClicked(); }
 
-void RunView::setRunEnabled(bool const enable) {
-  m_uiForm.pbRun->setText(enable ? "Run" : "Running...");
-  m_uiForm.pbRun->setEnabled(enable);
+void RunView::setRunEnabled(bool const enable) { setRunText(enable ? "Run" : "Running..."); }
```

Comment:
Given that functionality relies on this `"Run"` string (on line 30), this might be better as a constant at the top of the file. 

Strictly speaking, this is also GUI logic so more suited to the presenter, but it's so minor and borderline that it's up to you, really.